index.d.ts

```diff
@@ -115,6 +115,32 @@ declare const open: {
 	```
 	*/
 	apps: Record<open.AppName, string | readonly string[]>;
+
+	/**
+	Open an application. Cross-platform.
```

Comment:
```suggestion
	Open an app. Cross-platform.
```

Use the word `app` in other places too.